corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -112,6 +111,18 @@ def _get_apps(self, restore_state, restore_user):
 
         if app_aware_sync_app:
             apps = [app_aware_sync_app]
+        elif (
+            toggles.RESTORE_ACCESSIBLE_REPORTS_ONLY.enabled(restore_user.domain)
+            and restore_state.params.is_webapps
+        ):
+            apps = []
+            if restore_user.request_user.can_view_apps():
```

Comment:
What would you think of moving this conditional into the `elif` above?  Feels like that might more clearly portray that the toggle only applies to web apps users who can't view apps, and everyone else gets the `else` clause below (so it's clear it's not just a coincidence that lines 121 and 127 are the same).

Reply:
Yeah I can get behind that. My rationale was to clearly show the ambiguous scenario where we can't confidently determine if the restore is from web apps or live preview, and fallback to the default behavior, but to your point it isn't clear that it falls back to the default behavior.